scripts/visualize/visualize.matthew_map.R

```diff
@@ -1,11 +1,13 @@
 visualize.matthew_map <- function(viz){
   
-  counties <- readData(viz[['depends']])
+  counties <- readData(viz[['depends']][1])
```

Comment:
what should we be doing for reading in multiple dependencies. what I did here ☝️ is fragile, but it works.